lib/pharos/up_command.rb

```diff
@@ -72,8 +72,10 @@ def load_terraform(file, config)
       tf_parser = Pharos::Terraform::JsonParser.new(File.read(file))
       config['hosts'] ||= []
       config['api'] ||= {}
+      config['addons'] ||= {}
       config['hosts'] += tf_parser.hosts
       config['api'].merge!(tf_parser.api) if tf_parser.api
+      config['addons'].merge!(tf_parser.addons) if tf_parser.addons
```

Comment:
Isn't the `if` completely unnecessary as `tf_parse.addons` will always return a hash. Yes, it'll be an empty hash if no addons in the output but empty hash is not falsey for the if

Reply:
> Isn't the if completely unnecessary

Yes.